aviary/docs/user_guide/outputs_and_how_to_read_them.md

```diff
@@ -79,6 +79,14 @@ OpenMDAO has a reports system which will generate reports when you run your mode
 
 There is an SQLite database output. By default, it is `aviary_history.db`. It can be used to rerun your case though we do not detail that here. Users can write separate Python script to create user customized outputs and graphs. We will show how to use the this database to create user's customized graph in [the onboarding docs](../getting_started/onboarding.md).
 
+### Report Location
+
+Reports are placed a subdirectory called `reports` under the directory where the script was run. The subdirectory's name for Level 1 runs is based on the name of the CSV file used to define aircraft and mission properties. For Level 2 and Level 3 runs, the name is based on the name of the Python script being run. 
+
+### Warning About the Reports Directory
+
+Repeated runs of the same script (Levels 2 and 3) or the same aircraft defintion CSV file (Level 1) will overwrite existing reports directories. So to preserve reports from previous runs, the user should make copies of the report directory before starting another run.
```

Comment:
should we include a warning (maybe something similar to [this](https://github.com/OpenMDAO/Aviary/blob/c8780f1835b036538be5dc706c82d1623520e40f/.github/install_env_from_github.sh#L37)) before starting the analysis?

Reply:
(doesn't have to be a part of this PR)